scheduler/src/cook/kubernetes/api.clj

```diff
@@ -82,16 +82,21 @@
   (when-not (synthetic-pod? pod-name)
     pod-name))
 
-(defn assoc-uuids
-  "Returns event-map with added fields :job-uuid (based on pod-name) and :instance-uuid (if pod is not synthetic)"
+(defn assoc-fields
+  "Returns event-map with added fields instance-uuid (if applicable), job-name, job-uuid, pool, and user"
   [event-map pod-name]
   (let [instance-uuid (pod-name->instance-uuid pod-name)
-        job-uuid (or (pod-name->job-uuid pod-name)
-                     (cached-queries/instance-uuid->job-uuid-cache-lookup instance-uuid))]
+        job-name (str (:job/name (cached-queries/pod-name->job-ent-cache-lookup pod-name)))
+        job-uuid (str (:job/uuid (cached-queries/pod-name->job-ent-cache-lookup pod-name)))
+        pool (str (:pool/name (:job/pool (cached-queries/pod-name->job-ent-cache-lookup pod-name))))
+        user (str (:job/user (cached-queries/pod-name->job-ent-cache-lookup pod-name)))]
```

Comment:
Can we extract a variable for `cached-queries/pod-name->job-ent-cache-lookup` since we call it 4 times? Then we can also potentially destructure the fields we need from the job. I think the `str` calls for job name, pool name, and job user are superfluous.